src/db/dynamoDbClient.ts

```diff
@@ -0,0 +1,22 @@
+import AWS from "aws-sdk";
+import { DataMapper } from "@aws/dynamodb-data-mapper";
+
+let config: any = {};
+const ACCESS_KEY_ID = process.env.ACCESS_KEY_ID
+const SECRET_ACCESS_KEY = process.env.SECRET_ACCESS_KEY
+
+if (process.env.NODE_ENV === "development") {
```

Comment:
How would you refactor this configuration to have it removed from the inner layers of the application (data-access) to the outer layers of the application?

Reply:
I would make this a ES6 class where the values of region, endpoint, accessKeyId, secretAccessKey would be part of the constructor or setter of the class. So upper layer would have the responsibility of setting these attributes depending on the targeting environment.